File: js/lifecycle.js
describe( "life cycle tests", function() {

  beforeEach( function() {
    console.log( "before each" );
  } )

  afterEach( function() {
    console.log( "after each" );
  } )

  it( "test1", function() {
    console.log( "test 1" );
  } )

  it( "test2", function() {
    console.log( "test 2" );
  } )

} )

File: java/lifecycle.java
import org.junit.*;

public class LifeCycleTests {

  @Before
  public void beforEach() {
    System.out.println( "before" ); 
  }

  @After
  public void afterEach() {
    System.out.println( "after" ); 
  }

  @Test
  public void test1() {
    System.out.println( "test1" ); 
  }

  @Test
  public void test2() {
    System.out.println( "test2" ); 
  }

}
